script.js

```diff
@@ -209,6 +203,12 @@ let configureDungeon = () => {
       return false;
     }
     verbose && console.log("Go to boss");
+    const event = new MouseEvent("click");
+    const bossTile = document.querySelector(".tile-boss, .tile-ladder");
+    if (bossTile) {
+      bossTile.dispatchEvent(event);
+      DungeonRunner.handleInteraction();
```

Comment:
You should return true here too.

Reply:
In fact, I shouldn't

The script will try to click on the tile every time, but sometimes, the click won't have any impact if the boss tile is still not clickable
in this case, we want the function to continue as if we had never clicked

Reply:
and yes, it should work with multi layer dongeon ;)